frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -7,7 +7,8 @@ import {
   legendColorScale,
   riskSorting,
   mapMetrics,
-  isInitialized,
+  isWaitingOrDisabled as _isWaitingOrDisabled,
+  isError as _isError,
```

Comment:
Avoid variable names with leading or trailing underscores. We want to get away from these in general in console and add an eslint rule that disables it in the future. I would call the evaluated values `waitingOrDisabled` and `error` below to avoid the name conflicts.

Reply:
@spadgett thanks for pointing it out. I will keep these lines in mind for the next patches. Right now I don't want to cancel your approval by pushing again :sweat_smile: